tests/console/system_prepare.pm

```diff
@@ -38,6 +38,18 @@ sub run {
         }
     }
 
+    # For media upgrade, need reregistration the modules after migration
```

Comment:
Comments should describe it is only workaround for SLE12+

Reply:
Added comments